diff --git a/kiero.h b/kiero.h
index 4aba3d4..d4d2b2d 100644
--- a/kiero.h
+++ b/kiero.h
@@ -8,10 +8,10 @@
 #define KIERO_INCLUDE_D3D9   0 // 1 if you need D3D9 hook
 #define KIERO_INCLUDE_D3D10  0 // 1 if you need D3D10 hook
 #define KIERO_INCLUDE_D3D11  0 // 1 if you need D3D11 hook
-#define KIERO_INCLUDE_D3D12  0 // 1 if you need D3D12 hook
+#define KIERO_INCLUDE_D3D12  1 // 1 if you need D3D12 hook
 #define KIERO_INCLUDE_OPENGL 0 // 1 if you need OpenGL hook
 #define KIERO_INCLUDE_VULKAN 0 // 1 if you need Vulkan hook
-#define KIERO_USE_MINHOOK    0 // 1 if you will use kiero::bind function
+#define KIERO_USE_MINHOOK    1 // 1 if you will use kiero::bind function
 
 #define KIERO_ARCH_X64 0
 #define KIERO_ARCH_X86 0
